slack-infra/resources/slack-moderator-words/filters.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: slack-moderator-words-filters
+data:
+  filters.yaml: |-
+    - triggers:
+      - guys
+      action: chat.postEphemeral
+      message: "May I suggest \"all\" instead when addessing a group of people? Thank you. :slightly_smiling_face:"
```

Comment:
```suggestion
      message: "May I suggest \"all\" instead of \"guys\" when addessing a group of people? Thank you. :slightly_smiling_face:"
```

Reply:
Done :+1: 

Also, confirming that the bot won't be triggered if it's a message created by the bot itself: https://github.com/kubernetes-sigs/slack-infra/blob/8525eb3de12c51a07444398ed1ba06e416d3b9ba/slack-moderator-words/events.go#L76-L79